.github/workflows/ci.yml

```diff
@@ -62,7 +62,7 @@ jobs:
           node-version: 16.x
       - name: Publish
         run: |
-          npx ovsx@0.5.1 publish -i artifacts/*/*.vsix -p ${{secrets.OPEN_VSX_TOKEN}}
+          npx ovsx@0.8.0 publish -i artifacts/*/*.vsix -p ${{secrets.OPEN_VSX_TOKEN}}
```

Comment:
Is it definitely worth specifying the version, or could we go with a versionless invocation and trust latest?

Reply:
0.8.0 has worked well for us, but I'm happy to go version-less.